src/main/puzzle/puzzle.gd

```diff
@@ -52,78 +56,14 @@ func show_detail_message(text: String) -> void:
 	$HudHolder/Hud.show_detail_message(text)
 
 
-"""
-Shows a succinct single-line message, like 'Game Over'
-"""
-func show_message(text: String) -> void:
-	$HudHolder/Hud.show_message(text)
-
-
-func start_game() -> void:
-	PuzzleScore.prepare_game()
-	show_message("3")
-	$ReadySound.play()
-	yield(get_tree().create_timer(0.8), "timeout")
-	show_message("2")
-	$ReadySound.play()
-	yield(get_tree().create_timer(0.8), "timeout")
-	show_message("1")
-	$ReadySound.play()
-	yield(get_tree().create_timer(0.8), "timeout")
-	$GoSound.play()
-	_go_voices[randi() % _go_voices.size()].play()
-	PuzzleScore.start_game()
-
-
-"""
-Ends the game. This occurs when the player loses, wins, or runs out of time.
-"""
-func end_game() -> void:
-	var delay: int = 2.2
-	var message: String
-	var sound: AudioStreamPlayer
-	if PuzzleScore.scenario_performance.lost:
-		sound = $GameOverSound
-		message = "Game over"
-	elif not PuzzleScore.milestone_met(Global.scenario_settings.success_condition):
-		sound = $MatchEndSound
-		message = "Finish!"
-	else:
-		sound = $ExcellentSound
-		message = "You win!"
-		delay = 4.2
-	
-	sound.play()
-	PuzzleScore.end_game()
-	show_message(message)
-	yield(get_tree().create_timer(delay), "timeout")
-	emit_signal("after_game_ended")
-
-
 func get_milestone_hud() -> MilestoneHud:
 	return $Chalkboard/MilestoneHud as MilestoneHud
```

Comment:
Give these methods like get_milestone_hud() a look to make sure they're still used. Many of them like `set_chalkboard_visible` were removed already but we may have overlooked some.

Reply:
Removed several unused delegate methods; `PuzzleHud.show_detail_message()`, `Puzzle.get_next_piece_displays()`, `Puzzle.show_detail_message()`, `Puzzle.get_milestone_hud()`, `Puzzle.set_block()`